lib/open_assets/blockchain_server.rb

```diff
@@ -1,12 +1,12 @@
 module OpenAssets
   class BlockchainServer
-    def create_btc_maintenance(product, destination, amount)
+    def create_btc_maintenance(product, amount)
       params = {
-        public_address: product.wallet_public_address,
-        private_key: product.wallet_private_key,
-        recipient_address: destination,
+        public_address: ENV['CENTRAL_ADDRESS_PUBLIC_ADDRESS'],
+        private_key: ENV['CENTRAL_ADDRESS_PRIVATE_KEY'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.